### DISCUSSION TITLE
Let's explain how this helps protects us

### DISCUSSION THREAD

**Discussion by chrisstpierre** [OpenSignLabs/OpenSign#135] [Q&A] [answered]
So you sign a document digitally. 2-year later someone is suing you. You you have a signed contract that 100% protets you from the lawsuit.
But the counterparty says  the contract is not genuine. They say you forged this recently forged the contract. 

How does OpenSign save me from this sirimcustance?

**Comment by andrew-opensignlabs** [comment, marked as answer]
@chrisstpierre Thanks for initiating the discussion. I believe many others might have similar questions, because of which we are putting together an extensive FAQ list which will be published soon.

How digital signatures protect you is - 
- a PDF agreement is finalized between 2 parties
- Party 1 signs the document at which point the document gets a green tick if you open it in Adobe Acrobat and becomes immutable meaning if anybody tries to modify the document after this, the signature becomes invalid.
- When party 2 receives the agreement which is already signed by party 1, they are only allowed to add their sign(without modifying any content) and the updated document with 2 signatures is made immutable. Both the parties will receive a copy of the immutable document with 2 signatures on their email.
- During this entire process an audit trail is generated which can be downloaded as completion certificate for added security which has the mention of all IP addresses from where the document was accessed by each party.

Checking the authenticity of the document is as simple as opening it in Adobe Acrobat or any other tool that supports signature verification and clicking verify signatures button. These tools can cryptographically check if the signatures are valid and the document is intact.

Let me know if you still have any other questions.
